en/Basic Math/magic_square.md

```diff
@@ -0,0 +1,65 @@
+# What is a Magic Square Puzzle?
+
+A magic square is defined as a square containing several distinct integers arranged so that the total or sum of the numbers is the same in every row, column, and main diagonal and usually in some or all of the other diagonals.
+
+# Magic Square Formula
+
+A magic square puzzle of the `n` order is an organization of `n2` numbers, usually unique integers, in a square. The `n` numbers in all rows, columns, and diagonals add up to the same constant. A magic square bears the integers from 1 to `n2`. The fixed sum in every row, column,and diagonal are known as the magic constant or the magic sum. It is represented by the M letter . The magic constant of a typical magic square depends entirely on the value of `n`. Thus, the value of the magic sum is calculated using the following formula:
+
+- M = `n(n2 + 1)/2`
```

Comment:
Are you sure that this shouldn't be `n(n+1)/2`? (Arithmetic progression for sum of numbers from 1 to n²)

Reply:
yes , it's n square there but i didn't know how to write n² but now i can copy it from you thank you